app/src/common/shared/com/igalia/wolvic/ui/widgets/NavigationBarWidget.java

```diff
@@ -277,7 +277,7 @@ private void updateUI() {
 
         mBinding.navigationBarNavigation.userFeedbackButton.setOnClickListener(v -> {
             v.requestFocusFromTouch();
-            mWidgetManager.openNewTabForeground(getResources().getString(R.string.feedback_link));
+            mWidgetManager.openNewTabForeground(getResources().getString(R.string.feedback_link, BuildConfig.VERSION_NAME, DeviceType.getDeviceTypeId()));
```

Comment:
I think nobody uses `getDeviceTypeId()` because it's mostly useless, it just refers to the flavour which does not add a lot of information unless you only have 1 device per flavour. I think it's better to use getType() and remove getDeviceTypeId()

Reply:
done